ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -154,12 +154,25 @@ function computeRecleanFeatures(StudyFolder, ...
         % structure idx
         CIFTI_LENGTH=length(CIFTI.cdata);
 
-        CORTEX_Left_idx_list = cifti_diminfo_dense_get_surface_info(CIFTI.diminfo{1}, 'CORTEX_LEFT').ciftilist;
```

Comment:
This is what the matlab compiler was complaining about?  And it can't handle `size(..., [list])` either?  Such weird restrictions...

Reply:
Yes, the size function in 2017 version cannot handle the list and can only use positive integer...


```matlab
>> volume=read_avw("/media/myelin/shared/HCPpipelines_test_rc/HCD0001305_V1_MR/MNINonLinear/Results/fMRI_CONCAT_ALL/fMRI_CONCAT_ALL_hp0.ica/filtered_func_data.ica/melodic_IC.nii.gz");
>> size(volume,[1,2,3])
Error using size
Dimension argument must be a positive integer scalar within indexing range.
 
>> size(volume,1)

ans =

    91

>> size(volume,2)

ans =

   109

>> size(volume,3)

ans =

    91
```

Reply:
The matlab compiler also thinks that `cifti_diminfo_dense_get_surface_info(CIFTI.diminfo{1}, 'CORTEX_LEFT')` is a function which cannot be indexed by dot...

Reply:
Ah, I didn't realize interpreted 2017b didn't support dimension lists in `size` either.

Reply:
The following should work in pretty much every matlab version:
```
sz = size(volume)
sz(1:3)
```